### PR TITLE
Guests only available after installing app

### PR DESCRIPTION
**Pull request by enbrnz**
Added information about guests:
- only available after installing app
- where to get app

Let me know if you need more for this PR

### CODE REVIEW

File: modules/admin_manual/pages/configuration/user/user_roles.adoc [hunk 1]
```diff
@@ -21,6 +22,7 @@ The following information is not an in-depth guide, but more of a high-level ove
 
 == Guest
 
+* Only available after installing the {guests-app-url}[Guests app]
```
Comment (mmattel) [inline modules/admin_manual/pages/configuration/user/user_roles.adoc:25]:
```suggestion
* The {oc-marketplace-url}/apps/guests[Guests app] is available on the ownCloud Marketplace. You must install and enable it first.
```
The attribute version you do is possible and valid, but we do the style here which eases proofreading if the stuff gets more complex.

File: modules/admin_manual/pages/configuration/user/user_roles.adoc [hunk 2]
```diff
@@ -3,6 +3,7 @@
 :toclevels: 1
 :files_pdfviewer-url: {oc-marketplace-url}/apps/files_pdfviewer
 :collabora-online-app-url: {oc-marketplace-url}/apps/richdocuments
+:guests-app-url: {oc-marketplace-url}/apps/guests
```
Comment (mmattel) [inline modules/admin_manual/pages/configuration/user/user_roles.adoc:6]:
```suggestion
```

**Comment by mmattel** [comment]
If you agree with the changes, commit them here and I will approve it.
Please also let us know if this needs backporting to 10.6 and 10.7 (which I believe so)

File: modules/admin_manual/pages/configuration/user/user_roles.adoc (hunk 1 above)
Comment (enbrnz) [reply modules/admin_manual/pages/configuration/user/user_roles.adoc:25]: I don't mind, I was just following how it was done for the other apps. I would recommend to change that as well then  ;)

**Comment by enbrnz** [comment]
> Please also let us know if this needs backporting to 10.6 and 10.7 (which I believe so)

It does

Can we get this finalized somehow? How can I help?

**Comment by mmattel** [comment]
Timebudget resets tomorrow 😄 Will do on monday

**Review by EParzefall — APPROVED** [review]
LGTM

**Comment by phil-davis** [comment]
Backports:
10.7 #3506 
10.6 #3507

**Pull request merged**